llama/cli.py

```diff
@@ -1,15 +1,90 @@
-from datetime import datetime
+import datetime
 
 import click
+from botocore.exceptions import ClientError
 
+from llama import credit_card_slips
+from llama.alma import Alma_API_Client
 from llama.s3 import S3
+from llama.ses import SES
+from llama.ssm import SSM
 
 
 @click.group()
 @click.pass_context
 def cli(ctx):
     ctx.ensure_object(dict)
-    ctx.obj["today"] = datetime.today()
+    ctx.obj["today"] = datetime.datetime.today()
+
+
+@cli.command()
+@click.option(
+    "--date",
+    help=(
+        "Optional date of exports to process, in 'YYYY-MM-DD' format. Defaults to "
+        "yesterday's date if not provided."
+    ),
+)
+@click.option(
+    "--source_email",
+    required=True,
+    help="The email address sending the credit card slips.",
+)
+@click.option(
+    "--recipient_email",
+    required=True,
+    multiple=True,
+    help="The email address receiving the credit card slips. Repeatable",
+)
+@click.option(
+    "--api_key_parameter",
+    required=True,
```

Comment:
Should these parameter options pull from ENV? See comment below on bash script.

Reply:
See above